### PR TITLE
Add the possibility of ignoring WIP PRs/MRs

### PR DESCRIPTION
**Pull request by mkosiarc**
implements functionality requested in #83

### CODE REVIEW

**Comment by mkosiarc** [comment]
@pbortlov @sidpremkumar please review

**Comment by mkosiarc** [comment]
@msimacek @amarza-rh could you please review

File: reviewrot/__init__.py [hunk 1]
```diff
@@ -261,3 +262,24 @@ def represent_none(self, _):
...
+        results (list): list of BaseReview instances
+
+    Returns:
+        res (list): list of BaseReview instances with WIP
+                    reviews removed
+    """
+
+    res = []
+    for result in results:
+        match = re.match(r'WIP:?|\[WIP\]', str(result.title))
```
Comment (sidpremkumar) [inline reviewrot/__init__.py]: Not testing for lowercase? I'm assuming that is still fine.

Comment (msimacek) [reply reviewrot/__init__.py]: FYI, this is the regex that gitlab is using: https://gitlab.com/gitlab-org/gitlab-ce/blob/master/app/models/merge_request.rb#L304. So it should also match 'WIP\s' and be case insensitive

Comment (amarza-rh) [reply reviewrot/__init__.py]: Isn't this too broad a match? Any string that contains `WIP` will match, including acronyms etc.

What about:
* `'^WIP:?|^\[WIP\]` - match the prefix
* `^[^A-Za-z0-9_]*WIP[^A-Za-z0-9_]` -- skips all non-alphanumeric characters until the first WIP and expects non-alphanumeric after
* `^[^A-Za-z0-9]*[Ww][Ii][Pp][^A-Za-z0-9_]` -- like the one above, but also case-insensitive

Comment (mkosiarc) [reply reviewrot/__init__.py]: my bad, thanks for the suggestions. I changed the regex to what gitlab is using.

**Comment by pbortlov** [comment]
@mkosiarc We support "Work in Progress" as it is specified in the [Gitlab - "Work In Progress"](https://docs.gitlab.com/ee/user/project/merge_requests/work_in_progress_merge_requests.html)

**Comment by amarza-rh** [comment]
👍

**Comment by pbortlov** [comment]
:+1:

**Pull request merged**